src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -76,10 +82,18 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 						errors: result.errors
 					}
 				});
+				editEvent.withAdditionalMeasurements({
+					'elapsedTime': new Date().getTime() - startTime
+				}).send();
 			},
 			error => {
 				this._notificationService.error(localize('tableDesigner.errorProcessingEdit', "An error occured while processing the change: {0}", error?.message ?? error));
 				this.updateState(this.valid, this.dirty);
+				this._adsTelemetryService.createErrorEvent(TelemetryView.TableDesigner, edit.type, error?.code, error?.message).withAdditionalProperties({
```

Comment:
Have you verified the message doesn't contain any potential PII like paths or filenames?